src/main/java/com/training/qlsv/utils/MenuUtils.java

```diff
@@ -0,0 +1,83 @@
+package com.training.qlsv.utils;
+
+import com.training.qlsv.service.CourseService;
+import com.training.qlsv.service.StudentService;
+
+import java.util.Scanner;
+
+public class MenuUtils {
+    static Scanner scanner = new Scanner(System.in);
+
+    public static void getMainMenu() {
+        System.out.println("Có 2 option");
+        System.out.println("1. Xem menu sinh viên");
+        System.out.println("2. Xem menu các khóa học");
+        System.out.println("3. Kết thúc chương trình");
+        int chooseMainMenu = scanner.nextInt();
+        if (chooseMainMenu == 1) {
```

Comment:
các giá trị số 1,2,3 định nghĩa là biến toàn bộ( global variables)

Reply:
đã chuyển thành enum